controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/user/GroupId.java

```diff
@@ -0,0 +1,116 @@
+package com.yahoo.vespa.hosted.controller.api.integration.user;
+
+import com.yahoo.config.provision.ApplicationName;
+import com.yahoo.config.provision.TenantName;
+import com.yahoo.vespa.hosted.controller.api.role.ApplicationRole;
+import com.yahoo.vespa.hosted.controller.api.role.ProtoRole;
+import com.yahoo.vespa.hosted.controller.api.role.Role;
+import com.yahoo.vespa.hosted.controller.api.role.Roles;
+import com.yahoo.vespa.hosted.controller.api.role.TenantRole;
+
+import java.util.Objects;
+
+/**
+ * An identifier for a group of {@link UserId}s, corresponding to a bound {@link Role}.
+ *
+ * @author jonmv
+ */
+public class GroupId {
```

Comment:
I don't understand why you're introducing group here. We're creating/deleting/updating roles, not groups. The fact that some backends might choose to use groups for implementing the functionality shouldn't leak into the api.

Reply:
As I said, no strong opinion on the name. `RoleId` sounds good?